ee/codegen/src/__test__/__snapshots__/workflow-sandbox.test.ts.snap

```diff
@@ -1,5 +1,39 @@
 // Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
+exports[`Workflow Sandbox > write > should generate DatasetRow with mocks when mocks are provided 1`] = `
+"from vellum.workflows.inputs import DatasetRow
+from vellum.workflows.sandbox import WorkflowSandboxRunner
+
+from .inputs import Inputs
+from .workflow import TestWorkflow
+
+dataset = [
+    DatasetRow(
+        label="Scenario with Mocks",
+        inputs=Inputs(test_input="test-value"),
+        mocks=[
+            {
+                "node_id": "550e8400-e29b-41d4-a716-446655440001",
+                "when_condition": {
+                    "type": "CONSTANT_VALUE",
+                    "value": {"type": "JSON", "value": True},
+                },
+                "then_outputs": {"result": "mocked_result"},
+            },
```

Comment:
DevinAI we should be creating `MockNodeExecution` instances that are imported from `vellum.workflows`